content/en/synthetics/cicd_integrations/_index.md

```diff
@@ -23,19 +24,19 @@ further_reading:
 
 ---
 
-In addition to running tests at predefined intervals, you can also run Datadog Synthetic tests on-demand using API endpoints. You can run Datadog Synthetic tests in your continuous integration (CI) pipelines, letting you block the deployment of branches that would break your product.
+In addition to running tests at predefined intervals, you can run Datadog Synthetic tests on-demand using API endpoints. Run Datadog Synthetic tests in your continuous integration (CI) pipelines to block branches from being deployed and breaking your application in production.
```

Comment:
We should definitely remove the reference to the API in the first sentence, could we change it to `you can run Datadog Synthetic tests on-demand using the datadog-ci package or our API`, and link to the datadog-ci package docs?